qa/workunits/rbd/krbd_watch_errors.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -ex
+set -o pipefail
+
+function refresh_loop() {
+    set +x
+
+    local i
+    for ((i = 1; ; i++)); do
+        echo 1 | sudo tee "${SYSFS_DIR}/${dev_id}/refresh" > /dev/null
```

Comment:
nit (may be ignored): for readability I would either pass `$dev_id` as the `refresh_loop` argument, or use uppercase to make clear it is a global.

Reply:
`dev_id` is now passed as an argument, thanks!